### PR TITLE
Include spaces in the list info

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
Spaces have been included in the list info below file lists that support displaying spaces.

## Related Issue

- Fixes https://github.com/owncloud/web/issues/7924

## Screenshot:
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

### CODE REVIEW

**Review by kulmann — CHANGES_REQUESTED** [review]
As discussed, this PR can't go into the stable branch because of changes in or additional translation strings. Up to you if you want to make two different PRs for stable and master, or only point this PR to master.

File: packages/web-app-files/src/components/FilesList/ListInfo.vue [hunk 1]
```diff
@@ -37,7 +48,8 @@ export default {
   },
   computed: {
     items() {
-      return this.files + this.folders
+      const filesAndFolderCount = this.files + this.folders
+      return this.showSpaces ? filesAndFolderCount + this.spaces : filesAndFolderCount
```
Comment (kulmann) [inline packages/web-app-files/src/components/FilesList/ListInfo.vue:52]: Why not single line as follows?
```js
return this.files + this.folders + (this.showSpaces ? this.spaces : 0)
```

Comment (kulmann) [reply packages/web-app-files/src/components/FilesList/ListInfo.vue:52]: And more radical: isn't `this.spaces` always `0` if `this.showSpaces` evaluates to `false`? In that case you could just do

```js
return this.files + this.folders + this.spaces
```

File: packages/web-app-files/src/store/getters.ts [hunk 2]
```diff
@@ -32,9 +32,11 @@ export default {
   totalFilesCount: (state, getters) => {
     const fileCount = getters.filesAll.filter((file) => file.type === 'file').length
     const folderCount = getters.filesAll.filter((file) => file.type === 'folder').length
+    const spaceCount = getters.filesAll.filter((file) => file.type === 'space').length
```
Comment (kulmann) [inline packages/web-app-files/src/store/getters.ts:35]: Is this sufficient? As far as I can tell we don't want to count personal spaces for example. Which would mean that you need to extend the filter callback to also check the driveType.

**Review by kulmann — CHANGES_REQUESTED** [review]
Rebase please :-) Found some small issues, please see comments.

File: packages/web-app-files/src/store/getters.ts (hunk 2 above)
Comment (JammingBen) [reply packages/web-app-files/src/store/getters.ts:35]: Good catch, t hx!

**Pull request merged**